README.md

```diff
@@ -36,11 +37,20 @@ by anyone. With the following tenets in mind.
 
 #### Planned Interceptors
 * Bandwidth Estimation
-  - [NADA](https://tools.ietf.org/html/rfc8698)
+  - [NADA][rfc8698]
 * JitterBuffer, re-order packets and wait for arrival
 * [FlexFec](https://tools.ietf.org/html/draft-ietf-payload-flexible-fec-scheme-20)
 * [webrtc-stats](https://www.w3.org/TR/webrtc-stats/) compliant statistics generation
-* [RTCP Feedback for Congestion Control](https://datatracker.ietf.org/doc/html/rfc8888) the standardized alternative to TWCC.
+* [RTCP Feedback for Congestion Control][rfc8888] the standardized alternative to TWCC.
+
+### RFCs
```

Comment:
I find it slightly confusing to have the list *planned*/*implemented* duplicated. Wouldn't it be enough to have the RFC numbers mentioned in the list above?

Reply:
I've tried to harmonize the Pion repos. And more of them are using a list of implemented/planned RFCs